### DISCUSSION TITLE
how to create APk without android studio using cap ?

### DISCUSSION THREAD

**Discussion by elycheikhsmail** [ionic-team/capacitor#4645] [Q&A]
I see in cap v3 presentation it is possible create web native app without instal native IDE but I don't how ? for android app as example

**Comment by jcesarmobile** [comment]
You can run apps without opening the IDE with `npx cap run android` command, but it builds a debug apk, there is no command for release apks.

**Reply by elycheikhsmail** [reply]
> You can run apps without opening the IDE with `npx cap run android` command, but it builds a debug apk, there is no command for release apks.

thank you, did I need to i nstall java befor this command ?

**Reply by jcesarmobile** [reply]
I don’t think so
